lib/bolt/cli.rb

```diff
@@ -453,15 +453,15 @@ def show_task(task_name)
     end
 
     def list_tasks
-      outputter.print_tasks(pal.list_tasks, pal.list_modulepath)
+      outputter.print_tasks(pal.list_tasks, pal.list_modulepath, options[:filter])
```

Comment:
It seems better to apply the filter _here_ (and ditto in `list_plans`) so that each outputter doesn't have to support it separately (especially since I can't imagine the behavior differing between outputters).